northstar/src/runtime/console.rs

```diff
@@ -339,8 +339,8 @@ where
     S: AsyncRead + Unpin,
 {
     let required_permission = match &request {
-        model::Request::ContainerStats { .. } => Permission::ContainerStatistics,
         model::Request::Containers => Permission::Containers,
+        model::Request::Inspect { .. } => Permission::Containers,
```

Comment:
Again the name clash. All of the permission match 1:1 a request type which is idiomatic.